packages/wonder-blocks-announcer/src/index.ts

```diff
@@ -0,0 +1,334 @@
+// TODO: publish wonder-blocks-style package WB-1776
+// import {srOnly} from "../../wonder-blocks-style/src/styles/a11y";
+import type {
+    PolitenessLevel,
+    RegionDef,
+    RegionDictionary,
+    RegionFactory,
+} from "./announcer.types";
+
+const TIMEOUT_DELAY = 5000;
+let announcer: Announcer | null = null;
+
+export type SendMessageProps = {
+    message: string;
+    level?: PolitenessLevel;
+    removalDelay?: number;
+};
+
+/**
+ * Public API method to send screen reader messages.
+ * @param {string} message The message to send.
+ * @param {PolitenessLevel} level Polite or assertive announcements
+ * @param {number} removalDelay Optional duration to remove a message after sending. Defaults to 5000ms.
+ * @returns {string} IDREF for targeted live region element
+ */
+export function sendMessage({
+    message,
+    level = "polite", // TODO: decide whether to allow other roles, i.e. role=`timer`
+    removalDelay,
+}: SendMessageProps): string {
+    announcer = Announcer.getInstance();
+
+    if (typeof jest === "undefined") {
+        setTimeout(() => {
+            return announcer?.announce(message, level, removalDelay);
+        }, 100);
```

Comment:
**question:** Is there a specific reason why we need to use this timeout? I wonder if there's another way to handle this case, but I don't think I have full context on that.

Reply:
It was for Safari and VoiceOver, but I can play around with it again to see if we really need it.

Reply:
Yes, this does seem important for Safari in particular! I made it configurable so it can be manipulated at runtime if necessary (especially for tests).

Reply:
I ended up removing this with the move to async! It wasn't needed in my testing for Safari/Voiceover.

Reply:
An initial timeout might still be needed to get reliable announcements! The async code does seem to be messing with Voiceover in both Chrome and Safari, especially on the first couple of messages. When I test in Storybook, polite messages are getting dropped almost entirely and only some assertive messages are making it through. So I'll figure out how to work it back in..probably in the `announce` method of the Announcer class.

Reply:
I did some more testing and added the initial timeout back in for Safari. I also included an option to set the length to 0 so that tests can skip it and work properly.